gnovm/tests/files/map1.gno

```diff
@@ -0,0 +1,14 @@
+package main
+
+func main() {
+	type s struct{}
+	m := map[s]int{
+		{}: 1,
+		{}: 2,
+	}
+	// NOTE(tb): surprisingly, this code doesn't trigger any error in go, whereas
+	// the resulting map has only one item (the last one).
```

Comment:
Let's open a PR on https://github.com/golang/go!

Reply:
I created an issue for now :)
https://github.com/golang/go/issues/60161